pom.xml

```diff
@@ -1067,7 +1093,7 @@
               </execution>
 
               <execution>
-                <phase>generate-sources</phase>
+                <phase>compile</phase>
```

Comment:
Thanks for the heads-up, Jesse - a bump to stage view will come, coupling it to v2.0 of the pipeline dependencies, and this will bite me when it does.